# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/InternalRequest.h
# Commit: 55f13fb4ee3f
# Full Hash: 55f13fb4ee3f06209d00fd91a7a03884bab88e2d
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-12-15 09:27:59
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement csp for Module Workers; r=evilpie,asuth,rpl,ckerschb
#   
#   Depends on D155691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156102
# ==============================================================================

diff -r 354711cf113a -r 55f13fb4ee3f dom/fetch/InternalRequest.h
--- a/dom/fetch/InternalRequest.h	Wed Dec 14 14:55:37 2022 +0000
+++ b/dom/fetch/InternalRequest.h	Wed Dec 14 14:55:38 2022 +0000
@@ -69,7 +69,7 @@
  *                   | TYPE_STYLESHEET
  * "track"           | TYPE_INTERNAL_TRACK
  * "video"           | TYPE_INTERNAL_VIDEO
- * "worker"          | TYPE_INTERNAL_WORKER
+ * "worker"          | TYPE_INTERNAL_WORKER, TYPE_INTERNAL_WORKER_STATIC_MODULE
  * "xslt"            | TYPE_XSLT
  * ""                | Default for everything else.
  *